docs/install.md

```diff
@@ -33,33 +33,35 @@ Select the following tabs for the installation description of your target.
 
     _Note_: Following this guide you could run the installation directly from a console via keyboard and screen, via SSH client or serial console.
 
-    ## 1. Download and extract the DietPi disk image
+    ## 1. Download and optionally extract the DietPi disk image
 
     Open [dietpi.com](https://dietpi.com/#download){: class="nospellcheck"}, select your SBC and click on **Download**. The disk image will be downloaded locally.
 
     _Example:_
     ![DietPi for Raspberry Pi download page](assets/images/DietPi-RaspberryPi-image.jpg "DietPi for Raspberry Pi download page"){: width="1186" height="561" loading="lazy"}
 
-    **Unzip the downloaded file to a local folder.**
+    The downloaded file is compressed (`.xz` file extension). Many flash tools (e.g. [balenaEtcher](https://www.balena.io/etcher/), [Rufus](https://rufus.ie/), [Raspberry Pi Imager](https://www.raspberrypi.com/software/)) can directly use this file format.  
+    Otherwise an appropriate extraction tool has to be installed (e.g. [7zip for Windows](https://www.7-zip.org/), [The Unarchiver (Macintosh)](https://theunarchiver.com/), [`xz-utils`](https://tukaani.org/xz/) (Linux)) to extract the `.img` file. They are free of charge and have been tested to decompress the image correctly (see below for extract instructions).
 
-    It is `xz`-compressed so you will need to install either [7zip for Windows](https://www.7-zip.org/) or [The Unarchiver (Macintosh)](https://theunarchiver.com/). Both are free of charge and have been tested to decompress the image correctly.
+    ??? hint "Extracting DietPi images (for flash tools not supporting the `.xz` file format)"
+        
+        1. Install the extraction tool (e.g. [7zip for Windows](https://www.7-zip.org/), [The Unarchiver (Macintosh)](https://theunarchiver.com/), [`xz-utils`](https://tukaani.org/xz/) (Linux)
+        1. Extract the downloaded `.xz` formatted file. This will generate the image file (file format`.img`).
 
-    Linux users will need to download and install `xz-utils`.
+        ??? hint "How to extract DietPi images on Linux (`xz-utils`)"
```

Comment:
Could be made a snippet as well, right? :)

Reply:
Also thought about that. 
But then the last sentence in the hint block would be identical for all:
_Replace `DietPi-Image.img.xz` with the correct name of the downloaded archive, e.g. `DietPi_VirtualBox-x86_64-Buster.ova.xz`. This will decompress the DietPi image file for you to use._

Actually, I changed the name for every installation (VirtualBox, VMware, UTM,...).
So, then I would name it Virtualbox (it is only an example).
Ok?

Reply:
Or we use `DietPi_*.xz` or so in the description, to indicate that it is a placeholder, and on Linux it blob-matches the actual file.